package.json

```diff
@@ -34,14 +34,25 @@
     "jest": "23.5.0",
     "jest-cli": "23.5.0",
     "lerna": "3.1.1",
-    "modular-css-core": "file:./packages/core",
+    "modular-css-aliases": "file:packages/aliases",
```

Comment:
It's not strictly necessary, of course, but I've gotten into the habit of specifying any local `file:` dependency that I publish in `dependencies`. I find it helps clarify what the root manifest is managing, and it has the benefit of making `npm prune --production` mildly useful (if you're, say, archiving the repo during a build and don't want `devDependencies` bloating things up, etc).

Reply:
I buy that as a useful distinction! Thanks for the advice, it's sorely needed 😅